ui/src/router/components/configuration/components/RouterConfigSection.js

```diff
@@ -1,19 +1,56 @@
-import React from "react";
-import { EuiFlexGroup, EuiFlexItem } from "@elastic/eui";
+import React, { useState } from "react";
+import { EuiButtonGroup, EuiFlexGroup, EuiFlexItem } from "@elastic/eui";
 import { ResourcesConfigTable } from "./ResourcesConfigTable";
 import { ConfigSectionPanel } from "../../../../components/config_section";
 import { RoutesConfigTable } from "./router_config_section/RoutesConfigTable";
+import { RulesConfigTable } from "./router_config_section/RulesConfigTable";
 
 export const RouterConfigSection = ({ config }) => {
+  // Toggle buttons
+  const [routesSelected, setToggleRoutesSelected] = useState("Routes");
+
+  const onChangeToggle = (optionId) => {
+    setToggleRoutesSelected(optionId);
+  };
+
+  const toggleButtons = [
+    {
+      id: "Routes",
+      label: 'Routes',
+    },
+    {
+      id: "Rules",
+      label: 'Rules',
+    },
+  ];
+
   return (
     <EuiFlexGroup direction="row" wrap>
       <EuiFlexItem grow={3}>
-        <ConfigSectionPanel title="Routes">
-          <RoutesConfigTable
-            routes={config.routes}
-            rules={config.rules}
-            defaultRouteId={config.default_route_id}
-          />
+        <ConfigSectionPanel
+          title={routesSelected}
+          toggle={
```

Comment:
I think 'toggle' is too specific for the reusable component. Rather than 'toggle' can we call it 'appendTitle' or something? Since we are passing a generic react component in, it shouldn't matter if it's a toggle or label or anything else.